.github/workflows/lintcommits.yml

```diff
@@ -28,6 +28,10 @@ jobs:
                 pattern: /^.{0,72}(?:\r?\n(?:(.{0,72})|(.*?([a-z]+:\/\/)?(([a-zA-Z0-9_]|-)+\.)+[a-z]{2,}(:\d+)?([a-zA-Z_0-9@:%\+.~\?&/=]|-)+).*?))*$/,
                 error: "Commit message lines are too long (maximum allowed is 72 characters, except for URLs)",
               },
+              {
+                pattern: /^Merge branch '[\w:]*' into \w*$/,
```

Comment:
I think more than just letters, numbers and underscores are allowed in branch names.
Feels like this could just be anything starting with `Merge branch` (Since it doesn't have a category it will be invalid anyway)

Reply:
Thanks for the review, should be good now!